common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4648,6 +4648,12 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     LLAP_LRFU_BP_WRAPPER_SIZE("hive.llap.io.lrfu.bp.wrapper.size", 64, "thread local queue "
         + "used to amortize the lock contention, the idea hear is to try locking as soon we reach max size / 2 "
         + "and block when max queue size reached"),
+    LLAP_CACHE_REPL_STRATEGY_CLASS("hive.llap.cache.repl.strategy.class", "", "Strategy class for managing the "
```

Comment:
Isn't this feature rather about the more general 'hydration' than replication?

Reply:
Fixed.